src/components/Drawer.jsx

```diff
@@ -102,10 +98,16 @@ const CloseButton = ({ onClose }) => {
 };
 
 Drawer.propTypes = {
-    isOpen: PropTypes.bool,
+    isOpen: PropTypes.bool.isRequired,
     title: PropTypes.oneOfType([PropTypes.arrayOf(PropTypes.node), PropTypes.node]),
-    content: PropTypes.oneOfType([PropTypes.arrayOf(PropTypes.node), PropTypes.node]),
+    content: PropTypes.oneOfType([PropTypes.arrayOf(PropTypes.node), PropTypes.node]).isRequired,
     onClose: PropTypes.func.isRequired,
     classNames: PropTypes.object,
     position: PropTypes.string,
 };
+
+Drawer.defaultProps = {
+    title: "",
+    classNames: "",
```

Comment:
classNames is object above, but you are passing string here. change this to empty object. Also, these values are already declared above, why do we need to redeclare here ?

Reply:
Lint throw erroros without it